### PR TITLE
Add content branch option to build flow

### PR DESCRIPTION
**Pull request by orius123**

### CODE REVIEW

File: build/build_content/add_cloudslang_content.sl [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+#
+#   The Apache License is available at
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+####################################################
+# Gets the cloud-slang-content repository.
+#
+# Inputs:
+#   - url - URL of content repository
+#   - target_dir - location to put content repository in
```
Comment (oritstone) [inline build/build_content/add_cloudslang_content.sl]: Missing inputs doc

File: build/build_content/add_cloudslang_content.sl [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+# Results:
+#   - SUCCESS
+#   - FAILURE
+####################################################
+namespace: build.build_content
+
+imports:
+  build_content: build.build_content
```
Comment (oritstone) [inline build/build_content/add_cloudslang_content.sl]: unneeded import of the same namespace

File: build/build_content/add_cloudslang_content.sl [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    - target_cli
+    - content_branch:
+        required: false
+  workflow:
+    - clone_slang_content:
+        do:
+          build_content.clone:
+            - url
+            - target_location: content_dir
+            - branch: content_branch
```
Comment (oritstone) [inline build/build_content/add_cloudslang_content.sl]: should be -  required: false

File: build/build_content/build_cli_flow.sl [hunk 4]
```diff
@@ -33,6 +33,9 @@ flow:
     - cloudslang_content_repo:
```
Comment (oritstone) [inline build/build_content/build_cli_flow.sl]: Add missing inputs to doc above

File: build/build_content/clone.sl [hunk 5]
```diff
@@ -26,9 +26,10 @@ flow:
   inputs:
```
Comment (oritstone) [inline build/build_content/clone.sl]: Do we want to use somehow our git clone flow from our content repo?

**Comment by orius123** [comment]
We can't. It uses ssh we are building locally.

**Comment by oritstone** [comment]
We also want to pack the changelog.md in the CLI

File: build/build_content/add_cloudslang_content.sl (hunk 2 above)
Comment (orius123) [inline build/build_content/add_cloudslang_content.sl]: wrong. I'm using an old version of cslang

**Comment by orius123** [comment]
it was merged as part of a different PR

**Comment by orius123** [comment]
@oritstone fixed.
please merge

**Pull request merged**